go-controller/pkg/ovn/master.go

```diff
@@ -712,6 +714,23 @@ func (oc *Controller) ensureNodeLogicalNetwork(nodeName string, hostSubnets []*n
 		}
 	}
 
+	// add the management IPs for this node to host network namespace's address set
+	hns := config.Kubernetes.HostNetworkNamespace
+	if hns != "" {
+		nsInfo, err := oc.waitForNamespaceLocked(hns)
+		if err != nil {
+			klog.Errorf("Failed to wait for namespace %s event (%v)",
+				hns, err)
```

Comment:
Did you copy that message from somewhere else? You're not really waiting for an "event"... In this case, since the admin has explicitly told us that this namespace should exist (via the config/command-line), then just "Failed to get namespace %q: %v" is probably good

Reply:
Fixed.